pagopa/io-app
# chore: [IOBP-1076] FAQ without CTA with sticky buttons

## Short description
This pull request includes changes to the `ZendeskSupportHelpCenter` component updating the logic for displaying footer actions without fixed option.

## List of changes proposed in this pull request
- Replace `actions` prop from the `IOScrollView` component with `FooterActions` component
- Adjust horizontal margin with `includeContentMargins={false}` and `ContentWrapper` component

## How to test
- Open FAQ
- Ensure that CTA buttons are at the end of the view

| Old | New |
|--------|--------|
| <img width="480" alt="Screenshot 2024-12-16 at 17 33 27" src="https://github.com/user-attachments/assets/78cb2888-4ef0-41a7-b5db-beeecdf0b30e" /> | <img width="483" alt="Screenshot 2024-12-18 at 11 21 50" src="https://github.com/user-attachments/assets/50f2484e-83d9-418d-8c6c-452d031ad708" />|
